build.rs

```diff
@@ -64,6 +64,16 @@ fn gather_data() -> (u32, String, u32, String) {
         let p_status_date = yaml["status"]["date"].as_str().unwrap();
         //println!("{} on {}; {:?}", p_status_state, p_status_date, p_domains);
 
+        // Get the "Preparations" paragraph from the markdown:
+        lazy_static! {
+            static ref RE_PREPS: Regex = Regex::new(r"(?s)## Preparations\n(.*)").unwrap();
```

Comment:
I'm not sure about this because comments can have hints to why something doesn't work as example bluewin.ch is geo-locked to switzerland...

Also we can't really trust the markdown content right now for example hotmail has information why it doesnt work and no `## Preperation` and a `## Delta` for config options.

Reply:
> Also we can't really trust the markdown content structure right now

I think that in this case, we should make the content structure more structured :wink: 

> bluewin.ch is geo-locked to switzerland

Yes, but it is unlikely that someone
- registers an account in Switzerland 
- only uses it there -> does not realize the geo-lock
- leaves Swizerland
- Tries to setup DC there

And they can still visit the website if they want more information. But within DC I would show as few information as necessary (both to keep the crate small and to keep it simple for users).

Does this make sense?

Reply:
yes, maybe you can make a structure rule that makes more sense and we can create a ci test that ensures every pr does respect it

Reply:
I am not sure if I understood you correctly but we could let the build fail if there is no "Preparations" section in one provider file.

Reply:
(this would be a one-line-change)

Reply:
If there is no section but the status.state is PREPERATION we should panic and fail the build

Reply:
Done, but now the build fails because of rogers.com.md.

Reply:
Can we include the whole markdown when status is broken?

Reply:
I did a commit for this